docs/testnet/limitations.md

```diff
@@ -0,0 +1,10 @@
+# Testnet Limitations
+The vision for Obscuro is to be fully compatible with the Ethereum Virtual Machine and provide an entirely decentralised privacy experience. Testnet goes a long way to achieving that but there is more to be done and for now there are some limitations. Over time the number of limitations will decrease.
+
+These are the key limitations to be aware of when developing for Obscuro testnet:
+
+1. Streaming events are not yet implemented. Instead events are only available in the transaction receipt.
+1. The gas feature is not implemented yet.
+1. Native value transfer is not implemented yet.
+1. New nodes cannot be added. For now the number of Obscuro nodes is fixed.
+1. Data revelation is not implemented yet.
```

Comment:
1. Security is not fully implemented. Some keys are still hardcoded.
2. The decentralised bridge is limited to two hardcoded ERC20 tokens.
3. The Layer 1 is currently a hosted network. In the next iteration we'll connect Obscuro to Ropsten.
4. The "Wallet Extension" is not fully polished yet.

Reply:
OK. But Goerli, not Ropsten